azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/NetworkInterface.java

```diff
@@ -39,36 +39,36 @@
     boolean isPrimary();
 
     /**
-     * @return the MAC Address of the network interface
+     * @return the Mac Address of the network interface
      */
     String macAddress();
 
     /**
      *
-     * @return the Internal DNS name assigned to this network interface
+     * @return the Internal Dns name assigned to this network interface
      */
     String internalDnsNameLabel();
 
     /**
      * Gets the fully qualified domain name of this network interface.
      * <p>
-     * a network interface receives FQDN as a part of assigning it to a virtual machine
+     * A network interface receives Fqdn as a part of assigning it to a virtual machine.
```

Comment:
even better, in javadocs, we could just spell it out as "fully qualified domain name (FQDN)"